app/uk/gov/hmrc/hec/models/hecTaxCheck/company/CTAccountingPeriod.scala

```diff
@@ -16,16 +16,62 @@
 
 package uk.gov.hmrc.hec.models.hecTaxCheck.company
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsObject, JsResult, JsValue, Json, OFormat}
+import uk.gov.hmrc.hec.models.hecTaxCheck.HECTaxCheckSource
 
 import java.time.LocalDate
 
-final case class CTAccountingPeriod(
-  startDate: Option[LocalDate],
-  endDate: LocalDate,
-  ctStatus: CTStatus
-)
+trait CTAccountingPeriod {
+  val endDate: LocalDate
+  val accountingTypePeriodFor: HECTaxCheckSource
+}
 
 object CTAccountingPeriod {
-  implicit val format: OFormat[CTAccountingPeriod] = Json.format
+
+  final case class CTAccountingPeriodDigital(
+    startDate: LocalDate,
+    endDate: LocalDate,
+    ctStatus: CTStatus
+  ) extends CTAccountingPeriod {
+    val accountingTypePeriodFor = HECTaxCheckSource.Digital
+  }
+
+  final case class CTAccountingPeriodStride(
+    startDate: Option[LocalDate],
```

Comment:
I think we can remove this field - we don't get a start date for the accounting period in the stride journey